### PR TITLE
[vcpkg-ci-ffmpeg, ffmpeg] Require CI pass on all triplets, fix dependencies

### PR DESCRIPTION
**Pull request by dg0yt**

### CODE REVIEW

**Comment by dg0yt** [comment]
arm64-android: Need `-lc++`, but some deps still need work:
mpg123: -lstdc++
modplug: .../libstdc++
x265: .../libc++

**Comment by cenit** [comment]
thanks. It was very difficult for me to accept to add failures for ffmpeg in my pr just because it was silently broken on master. I will make mine depend on this

File: ports/libmodplug/portfile.cmake [hunk 1]
```diff
@@ -13,7 +13,7 @@ vcpkg_from_github(
         ${STATIC_PATCH}
         002-detect_sinf.patch
         003-use-static-cast-for-ctype.patch
-        004-export-pkgconfig.patch  # https://github.com/Konstanty/libmodplug/pull/59
+        004-export-pkgconfig.patch
```
Comment (WangWeiLin-MV) [inline ports/libmodplug/portfile.cmake:16]: Since the content of the patch has changed, it is recommended to commit it to upstream for tracking.

**Review by WangWeiLin-MV — APPROVED** [review]
The ports `ffmpeg`, `libmodplug`, `libopenmpt` installation tests pass with the following triplets:
* x64-windows

File: ports/libmodplug/portfile.cmake (hunk 1 above)
Comment (dg0yt) [reply ports/libmodplug/portfile.cmake:16]: Basically yes. OTOH upstream didn'r accept the original PR in three years.

**Pull request merged**